mail_notification_custom_subject/models/mail_message_custom_subject.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, fields, models
+
+
+class MailMessageCustomSubject(models.Model):
+    _name = "mail.message.custom.subject"
+    _description = "Mail Message Custom Subject"
+
+    name = fields.Char(string="Template Name")
+    model_id = fields.Many2one(
+        comodel_name="ir.model",
+        string="Model",
+        required=True,
+        help="Model where this template applies",
+    )
+    subtype_ids = fields.Many2many(
+        comodel_name="mail.message.subtype", string="Applied Subtypes", required=True,
+    )
+    subject_template = fields.Char(
+        string="Subject Template",
+        required=True,
+        help="Subject (placeholders may be used here)",
+    )
+    position = fields.Selection(
+        selection=[
+            ("append_before", _("Append Before")),
+            ("append_after", _("Append After")),
+            ("replace", _("Replace")),
```

Comment:
This is a bug:

```suggestion
            ("append_before", "Append Before"),
            ("append_after", "Append After"),
            ("replace", "Replace"),
```

Never use `_()` outside of a method code. Otherwise, the translation is done without a real context, and becomes English always.

Reply:
Should you move the declaration of the selection fields to a method, then? @CarlosRoca13

Reply:
I think that's not necessary. @joao-p-marques

Reply:
Selection values inside a selection field are already translatable and handled by Odoo Framework without anything extra.